drivers/overlay/composefs_supported.go

```diff
@@ -77,6 +124,11 @@ func generateComposeFsBlob(toc []byte, destFile string) error {
 	if err != nil {
 		return err
 	}
+
+	if err := enableVerity("manifest file", newFd); err != nil {
+		logrus.Warningf("%s", err)
```

Comment:
I don't think we should warn here if we get e.g. `EOPNOTSUPP` - that'd just be annoying for everyone using filesystems which don't yet support it (like xfs, but that's in progress).

Reply:
yes good idea.  I've dropped that